src/main/java/counter/api/BookingsApiController.java

```diff
@@ -12,20 +25,51 @@
 
     private final HttpServletRequest request;
 
+    @Autowired
+    BookingService bookingService;
+
     @org.springframework.beans.factory.annotation.Autowired
     public BookingsApiController(ObjectMapper objectMapper, HttpServletRequest request) {
         this.objectMapper = objectMapper;
         this.request = request;
     }
 
+    // @Override
+    // public Optional<ObjectMapper> getObjectMapper() {
+    //     return Optional.ofNullable(objectMapper);
+    // }
+
+    // @Override
+    // public Optional<HttpServletRequest> getRequest() {
+    //     return Optional.ofNullable(request);
+    // }
+
+    @Override
+    public ResponseEntity<Void> bookingsBookingIdDelete(@ApiParam(value = "",required=true) @PathVariable("bookingId") Integer bookingId) {
+        Integer result = bookingService.deleteBooking(bookingId);
+        return new ResponseEntity<Void>(HttpStatus.valueOf(result));
+    }
+
     @Override
-    public Optional<ObjectMapper> getObjectMapper() {
-        return Optional.ofNullable(objectMapper);
+    public ResponseEntity<Void> bookingsPost(@ApiParam(value = "" ,required=true )  @Valid @RequestBody Booking body) {
+        try {
+            bookingService.saveBooking(body);
+            return new ResponseEntity<Void>(HttpStatus.CREATED);
```

Comment:
is this needed to set the status code, if so what is the annotations doing in the implented interfaces